### ISSUE TITLE
Allowing name to be edited after submitting an entry

### ISSUE THREAD

**Issue by januweness** [fsinfuhh/Bitpoll#66]
This is a great tool in place of doodle and offers a lot of flexibility.

A suggestion: As people make typos, it is good to have the edit function with which entries can be modified. As hard as it may be to believe, people even mis-type their names, and it would be useful to allow changing the name as well.

**Comment by JSquar** [comment]
This is already possible. The creator of the poll can allow changes to existing votes without authentification (this is disabled by default). Or if a user assigns their vote to themself they can also edit this vote afterwards.

**Comment by DerBiasto** [comment]
With this setting I can edit the votes, but I am unable to edit the names in this way.

**Comment by JSquar** [comment]
You're right. I was confused, I just focused on editing the vote itself but not the name.
